### PR TITLE
Add Tree Condition and CLE fields. Add tree benefit API.

### LINKED ISSUE #70: V2 - Tree Benefits Calculator - Add two new REQUIRED fields in 'Add Data'
1. Add two new REQUIRED fields - with drop down menus - right after the 'DBH (in)' entry screen.

2. First field is called "Tree Condition"  Drop down menu has the following values:

"Excellent"
"Good"
"Fair"
"Poor"
"Critical"
"Dying"
"Dead"

3. Second field is called "Crown Light Exposure (CLE)" Available choices in the drop down menu are:

"Fully Shaded"
"Partially Shaded"
"Fully Exposed to Light"

*These fields will be mapped and saved as numbers on the backend/firebase as*:

"Fully Shaded"  = 0
"Partially Shaded" = 3
"Fully Exposed to Light" = 5

### PR DESCRIPTION
**Pull request by jeremydavidson**
Added Tree Condition and CLE fields. Added tree benefit API.

Fixed #70

### CODE REVIEW

File: src/lib/firebaseServices/addTree.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import { TreeValidationTypes } from './../treeData';
+/* eslint-disable @typescript-eslint/camelcase */
```
Comment (halmueller) [inline src/lib/firebaseServices/addTree.ts:1]: Nice!

File: src/screens/AddTreeScreen/TreeTypeSelect.tsx [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-import { colors } from '../../styles/theme'
+import { Subheading, Button, useTheme } from 'react-native-paper'
 import { TreeTypes } from '../../lib/treeData'
 import { TtypeHelp } from './TtypeHelp'
 const coniferImage = require('../../../assets/conifer.png')
 const deciduousImage = require('../../../assets/deciduous.png')
 
 interface TreeTypeSelectProps {
-  onSelect: (speciesData: String) => void
+  onSelect: (speciesData: string) => void
```
Comment (halmueller) [inline src/screens/AddTreeScreen/TreeTypeSelect.tsx:11]: Question from a JS idiot: why lowercase string?

**Review by halmueller — APPROVED** [review]
LGTM. Some nice formatting cleanup along the way.

File: src/screens/AddTreeScreen/TreeTypeSelect.tsx (hunk 2 above)
Comment (jeremydavidson) [reply src/screens/AddTreeScreen/TreeTypeSelect.tsx:11]: It's a TypeScript primitive in lowercase, eslint probably auto-fixed this for us: https://www.typescriptlang.org/docs/handbook/basic-types.html.

**Pull request merged**